build.gradle.kts

```diff
@@ -63,24 +63,24 @@ allprojects {
     group = "com.github.triplet.gradle"
 
     afterEvaluate {
-        convention.findByType<JavaPluginExtension>()?.apply {
-            sourceCompatibility = JavaVersion.VERSION_1_8
+        extensions.findByType<JavaPluginExtension>()?.apply {
+            toolchain.languageVersion.convention(JavaLanguageVersion.of(11))
```

Comment:
Gradle 8.1.1 seems to need AGP 8 (I've seen some `NoSuchMethodError` in the logs related to CC) and AGP 8 requires a minimum of JDK11. Actually, the plugin was already using [JDK11 API (readNBytes) ](https://github.com/Triple-T/gradle-play-publisher/blob/master/play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/Bootstrap.kt#L187) but declared that it can run on JDK8.